docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/delete-edge-syntax.md

```diff
@@ -6,4 +6,12 @@ The `DELETE EDGE` statement is used to delete edges. Given an edge type, the sou
 DELETE EDGE <edge_type> <vid> -> <vid>[@<ranking>] [, <vid> -> <vid> ...]
 ```
 
+For example,
+
+```ngql
+nebula> DELETE EDGE follow 100 -> 200;
+```
+
+The above query deletes an edge whose source vertex is `100`, dest vertex is `200`, and the edge type is `follow`.
```

Comment:
`Ranking` is 0 default.

Reply:
I think it's ok here.

Reply:
This only delete the edge which ranking is 0.

Reply:
> This only delete the edge which ranking is 0.

If so, the behavior is changed from origin design.
BY DEFAULT, this sentence should delete all rankings instead of ranking == 0.
Otherwise, how can the user list the ranking range?? is there any sentence to list all the rankings?

Reply:
> > This only delete the edge which ranking is 0.
> 
> If so, the behavior is changed from origin design.
> BY DEFAULT, this sentence should delete all rankings instead of ranking == 0.
> Otherwise, how can the user list the ranking range?? is there any sentence to list all the rankings?

In Current Implementation, the delete edge sentence could only delete the edge with specific ranking(default 0).
If it don't fit the usage, we need submit a feature request.